Deprecating config entry listener with reloading methods in config flow (#3082)
home-assistant/developers.home-assistant: blog/2026-05-07-config-entry-listener-together-with-reloading-methods.md

---
author: G Johansson
authorURL: https://github.com/gjohansson-ST
authorImageURL: https://avatars.githubusercontent.com/u/62932417?v=4
authorTwitter: GJohansson

> title: "The use of config entry listener with reloading methods in config flow"

Reviewer: ```suggestion
title: "Deprecating config entry listener with reloading methods in config flow"
```

---

> As of Home Assistant Core 2025.6, using a config entry listener together with any reloading methods in a config flow is deprecated and will result in an error from 2025.12.

Reviewer: ```suggestion
As of Home Assistant Core 2026.6, using a config entry listener together with any reloading methods in a config flow is deprecated and will result in an error from 2026.12.
```
